src/company_cmd.cpp

```diff
@@ -574,14 +574,14 @@ restart:;
 void ResetCompanyLivery(Company *c)
 {
 	for (LiveryScheme scheme = LS_BEGIN; scheme < LS_END; scheme++) {
-		c->livery[scheme].in_use  = 0;
+		c->livery[scheme].in_use.Reset({Livery::Flag::Primary, Livery::Flag::Secondary});
```

Comment:
Aren't there helper functions that do a "full reset" rather than having to specify all enum values? Similarly for Any & All

Reply:
I've changed this (and the one near it) to be just `.Reset()` to ensure everything is reset.

I decided to be explicit with Primary/Secondary everywhere so that it's possible to later add more flags to the enum and still have things work.